openapi/code/method.go

```diff
@@ -46,9 +46,27 @@ type Method struct {
 	wait       *openapi.Wait
 	pagination *openapi.Pagination
 	Operation  *openapi.Operation
+	DataPlane  *openapi.DataPlane
 	shortcut   bool
 }
 
+// Returns the fields which contains the DataPlane info. Each field is nested in the previous one.
```

Comment:
[nit] Starts the comment with the name of the function:

```go
// DataPlaneInfoFields returns the fields ...
``` 

Reference: https://go.dev/wiki/CodeReviewComments#comment-sentences

Note: could you also do this for the other new functions in this PR (e.g. `HasDataPlaneAPI`)?